src/ui/components/GeneralSettings.tsx

```diff
@@ -138,6 +138,10 @@ export const GeneralSettings = () => {
                 label: 'Standard (W3C draft)',
```

Comment:
If can make a suggestion here - This label will be out of date as soon as the draft is further updated or a new version comes out. Since Figma does not allow different versions of the plugin, it's not clear what the 'default' of the dropdown will be, and might break someone's current StyleDictionary workflow when updated.
I don't think the drafts even have a version yet, but you could use a date?
`label: 'Standard (W3C Draft - 2022.08.31)'`